Add warning about plugin permission (#4383)
opensearch-project/documentation-website: _install-and-configure/plugins.md

You can install individual plugins for OpenSearch based on your needs. For information about available plugins, see [Available plugins](#available-plugins).

> ## Considerations for Plugins

Reviewer: ```suggestion

```

> In OpenSearch Plugins have the ability to access the data in the cluster, including metadata about cluster operations. Take care when selecting and installing plugins - only use those from trusted sources. Ensuring the plugin's source is reliable helps to mitigate risks to cluster's data and integrity. Always vet the source and understand the function of a plugin before installing on an OpenSearch cluster.

Reviewer: ```suggestion
OpenSearch plugins give you the ability to access the data in the cluster, including metadata about cluster operations. To help mitigate risks to your cluster's data and integrity,  make sure that a custom plugin's source is reliable when selecting and installing custom OpenSearch plugins. Furthermore, make sure that you understand the function of a plugin before installing on an OpenSearch cluster.
```
